src/domain/usecase/link-social-account.test.ts

```diff
@@ -0,0 +1,53 @@
+import { UserRepository } from '../..//application/repository'
```

Comment:
There's an unwanted additional slash:

```js
import { UserRepository } from '../../application/repository'
```

Was this the cause of the false negative you had with the `import/no-unresolved` rule?